react-youtube-adaptive-loading/src/store/api/youtube-api.js

```diff
@@ -1,143 +1,166 @@
-export function buildVideoCategoriesRequest() {
-    return buildApiRequest('GET',
-      '/youtube/v3/videoCategories',
-      {
-        'part': 'snippet',
-        'regionCode': 'US'
-      }, null);
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const buildVideoCategoriesRequest = () => {
+  return buildApiRequest('GET',
+    '/youtube/v3/videoCategories', {
+      'part': 'snippet',
+      'regionCode': 'US'
+    },
+    null
+  );
+};
+
+export const buildMostPopularVideosRequest = (amount = 12, loadDescription = false, nextPageToken, videoCategoryId = null) => {
```

Comment:
It feels like if we're standardizing on a central configuration for values that we should also have the `amount` here in the config?

Reply:
Yes, I think it should be centralized in the config.
Let me extract them and put into one config file.

Reply:
Cool. I'll await a commit that includes the amount in the centralized config and we can then merge this in.

Reply:
Sure, let me update them shortly.

Reply:
@addyosmani 

The config centralization is committed.